src/category_theory/category/Quiv.lean

```diff
@@ -0,0 +1,145 @@
+/-
+Copyright (c) 2021 Scott Morrison. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Scott Morrison
+-/
+import category_theory.path_category
+import category_theory.category.Cat
+
+/-!
+# The category of quivers
+
+The category of (bundled) quivers, and the free/forgetful adjunction between `Cat` and `Quiv`.
+
+-/
+
+universes v u
+
+namespace category_theory
+
+/-- Category of quivers. -/
+def Quiv := bundled quiver.{(v+1) u}
+
+namespace Quiv
+
+instance : has_coe_to_sort Quiv :=
+{ S := Type u,
+  coe := bundled.α }
+
+instance str (C : Quiv.{v u}) : quiver.{(v+1) u} C := C.str
+
+/-- Construct a bundled `Quiv` from the underlying type and the typeclass. -/
+def of (C : Type u) [quiver.{v+1} C] : Quiv.{v u} := bundled.of C
+
+instance : inhabited Quiv := ⟨Quiv.of (quiver.empty pempty)⟩
+
+/-- Category structure on `Quiv` -/
+instance category : large_category.{max v u} Quiv.{v u} :=
+{ hom := λ C D, prefunctor C D,
+  id := λ C, prefunctor.id C,
+  comp := λ C D E F G, prefunctor.comp F G,
+  id_comp' := λ C D F, by cases F; refl,
+  comp_id' := λ C D F, by cases F; refl,
+  assoc' := by intros; refl }
+
+/-- The forgetful functor from categories to quivers. -/
+@[simps]
+def forget : Cat.{v u} ⥤ Quiv.{v u} :=
+{ obj := λ C, Quiv.of C,
+  map := λ C D F,
+  { obj := λ X, F.obj X,
```

Comment:
Would `F.to_prefunctor` be better here? Or maybe that's harder to work with?

Reply:
Yes, that's great, thanks.